src/platform-includes/session-replay/install/javascript.mdx

```diff
@@ -1,13 +1,13 @@
-Install the Replay package with the package manager of your choice. Alternatively, you can load the Replay integration using a CDN bundle:
+The Replay Integration is **already included** in your Browser or Framework SDK NPM packages. If you're using CDN bundles instead of NPM packages, you need to load the Replay integration CDN bundle in addition to your browser bundle:
 
-```bash {tabTitle: ESM}
+```bash {tabTitle: NPM}
 # Make sure to have @sentry/browser or a Framework SDK (e.g. @sentry/react) installed
+npm install --save @sentry/browser
+```
 
-# Using yarn
-yarn add @sentry/replay
-
-# Using npm
-npm install --save @sentry/replay
+```bash {tabTitle: Yarn}
+# Make sure to have @sentry/browser or a Framework SDK (e.g. @sentry/react) installed
+yarn add @sentry/browser
```

Comment:
Do we want to keep these tabs or is it safe to assume that users who are on this page already installed an SDK package?

(In case we keep it, I'd vote to make framework-specific versions in a follow-up PR to show them the correct package installation instructions, just like in the wizard)

Reply:
I think we should keep, it may be useful for new users that are signing up for replay?